Packs/CadoResponse/pack_metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "Cado Response",
+    "description": "Automate data collection. Process data at cloud speed. Analyze with purpose.",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Cado Security",
+    "url": "https://docs.cadosecurity.com/docs/cado-response/integrations/cortex-xsoar/",
+    "email": "contact@cadosecurity.com",
+    "categories": [
+        "Forensics & Malware Analysis"
+    ],
+    "tags": [
+        "Forensics",
+        "Alerts",
+        "PII",
+        "Security",
+        "New",
+        "Trending"
+    ],
+    "useCases": [],
+    "keywords": [],
```

Comment:
to help users to be able to search packs based on vendor names, its preferred to add the vendor name as a keyword. (no obligation)